src/DisplayBoard.cpp

```diff
@@ -0,0 +1,110 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    using u_int = unsigned int;
+
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const u_int width, const char sign = PADDING);
+    std::string drawEndLine(const u_int width);
+    std::string drawEmptyLine(const u_int width, const char sign = PADDING);
+    std::string drawColumns(const u_int maxElementsInRows, const u_int colNumber, const u_int heightCol);
+    std::string drawRow(const u_int maxElementsInRows);
+    std::stringstream drawBoard(const u_int width,
```

Comment:
Consider making drawBoard parameterless or taking only Board reference as param.

Reply:
Imo, this is helper method and it should not be dependent in any way from Object. It should only take params as was described in 2.2. 
@LordLukin what do you think

Reply:
Think about usability. In this form you would pass parameters like 
`drawBoard(board.getWidth(), board.getHeight(), ...)`
So it will be easier to pass whole Board as a const &, because then you will only have to write
`drawBoard(board);`

Reply:
Of course I can change it, but: 
This method is hidden in "private section". It can not be called directly and IMO it should not be dependent from object. 
What do you think about it?

Reply:
done